README.md

```diff
@@ -71,6 +75,8 @@ In order to run with a `electrum` compatible server as the backend, in addition
       ELECTRUM_TLS_ENABLED: "false"
 ```
 
+## Configuration overrides
```

Comment:
Adding a new heading here to make it visually clear that this is a separate section.